data/default_release_images.json

```diff
@@ -35,5 +35,17 @@
         "cpu_architecture": "arm64",
         "url": "quay.io/openshift-release-dev/ocp-release:4.10.15-aarch64",
         "version": "4.10.15"
+    },
+    {
+        "openshift_version": "4.11",
+        "cpu_architecture": "x86_64",
+        "url": "quay.io/openshift-release-dev/ocp-release-nightly:4.11.0-0.nightly-2022-05-25-193227",
+        "version": "4.11.0-0.nightly-2022-05-25-193227"
```

Comment:
I think we should mark it as: support_level:beta

Reply:
I can do that
There's a logic to set ``support_level: beta`` if it detects release as a pre-release, but explicit is better than implicit

Reply:
> I can do that There's a logic to set `support_level: beta` if it detects release as a pre-release, but explicit is better than implicit

right, we detect it here: https://github.com/openshift/assisted-service/blob/4dae0b226c1dde6febb15d884449c6713fb854ce/internal/versions/versions.go#L380

Reply:
sorry, forgot to handle it
just a second

Reply:
done